src/main/java/att/grappa/ExceptionDisplay.java

```diff
@@ -1,202 +1,23 @@
-/*
```

Comment:
Why did you remove the license header? Can you put it back please?

Reply:
Actually the ExceptionDisplay.java is renamed to AwtExceptionDisplay.java. So this ExceptionDisplay.java is a new file, where the author is me.

Reply:
This still needs a license header, otherwise it might be assumed that the file is uncopyrighted, i.e. "public domain". I merged this PR, so I assumed that you're OK to include this under the same license as every other file in the project.